src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const tableHead = document.querySelector('thead');
+const cloneTableHead = tableHead.cloneNode(true);
+const tableFoot = document.querySelector('tfoot');
+const cloneTableFoot = tableFoot.cloneNode(true);
+const headPosition = cloneTableHead.querySelector('tr :nth-child(2)');
+const footPosition = cloneTableFoot.querySelector('tr :nth-child(2)');
+const headSalary = tableHead.querySelector('tr :nth-child(5)');
+const footSalary = tableFoot.querySelector('tr :nth-child(5)');
```

Comment:
You are selecting elements from the DOM and cloning them immediately. This can be inefficient if the elements are not used. Consider moving the cloning operation to the point where the clones are actually needed.